.github/workflows/tests.yml

```diff
@@ -0,0 +1,17 @@
+name: Tests
+
+on:
+  pull_request:
+    branches:
+      - staging
+    paths-ignore:
+      - '**.md'
```

Comment:
Shouldn't this also run on push to staging?

Reply:
Added